packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js

```diff
@@ -0,0 +1,113 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { View } from 'react-native';
+
+import ActionSheet from 'react-native-actionsheet';
+import {
+  Avatar,
+  BodyText,
+  Card,
+  CardContent,
+  H3,
+  H6,
+  PaddedView,
+  styled,
+  Touchable,
+  ButtonLink,
+  ChannelLabel,
+} from '@apollosproject/ui-kit';
+
+const StyledCard = styled(() => ({
+  marginHorizontal: 0,
+  marginVertical: 0,
+}))(Card);
+
+const EllipsisView = styled(({ theme }) => ({
+  paddingHorizontal: theme.sizing.baseUnit,
+}))(View);
+
+const GreyH3 = styled(({ theme }) => ({
+  color: theme.colors.text.tertiary,
+  textAlign: 'right',
+}))(H3);
+
+const GreyH6 = styled(({ theme }) => ({
+  color: theme.colors.text.tertiary,
+}))(H6);
+
+const StyledCardContent = styled(() => ({
+  alignItems: 'center',
+}))(CardContent);
+
+const StyledBodyText = styled(() => ({
+  textAlign: 'center',
+}))(BodyText);
+
+const BodyTextView = styled(({ theme }) => ({
+  marginTop: theme.sizing.baseUnit * 1.5,
+  height: theme.sizing.baseUnit * 16,
+  alignItems: 'center',
+}))(View);
+
+class PrayerCard extends PureComponent {
+  handleShowActionSheet = () => {
+    this.ActionSheet.show();
+  };
+
+  render() {
+    const { imageSource, name, text, id, flagRequest, source } = this.props;
+
+    const cancelIndex = 1;
+    const destructiveIndex = 0;
+    const handleOnPress = (index) => {
+      if (index !== cancelIndex) {
+        return flagRequest(id);
+      }
+      return index;
+    };
+    const options = ['Report as Inappropriate', 'Cancel'];
```

Comment:
these should really be props so we can add functionality to this card but we can do that in a different PR